doc/nrf/ug_thread_configuring.rst

```diff
@@ -249,7 +249,7 @@ Minimal Thread Device (MTD)
   Set :kconfig:`CONFIG_OPENTHREAD_MTD` to configure the device as MTD.
 
   By default, the MTD operates as Minimal End Device (MED).
-  To make it operate as Sleepy End Device (SED), enabling :kconfig:`CONFIG_OPENTHREAD_MTD_SED`.
+  To make it operate as Sleepy End Device (SED), enable :kconfig:`CONFIG_OPENTHREAD_MTD_SED`.
```

Comment:
I was actually wondering if this should be "set" since the other Kconfigs in the section use it. Opinions?

Reply:
I think "set" is more appropriate since other Kconfigs in the section use it as well.